pkgs/by-name/re/reprepro/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  autoreconfHook,
+  bzip2,
+  db,
+  gpgme,
+  libarchive,
+  xz,
+  zlib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "reprepro";
+  version = "5.4.7";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "debian";
+    repo = "reprepro";
+    tag = "reprepro-${version}";
+    hash = "sha256-/qiN79bq5uSCyaRnsBsrFDoTRIac3Pr5JOawkkl3cSk=";
+  };
+
+  buildInputs = [
+    bzip2
+    db
+    gpgme
+    libarchive
+    xz
+    zlib
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
+
+  meta = with lib; {
+    homepage = "https://salsa.debian.org/debian/reprepro/";
+    description = "Debian package repository producer";
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ baloo ];
+    platforms = platforms.all;
+    mainProgram = "reprepro";
+  };
+}
```

```suggestion
  meta = {
    homepage = "https://salsa.debian.org/debian/reprepro/";
    description = "Debian package repository producer";
    license = lib.licenses.gpl2;
    maintainers = with lib.maintainers; [ baloo ];
    platforms = lib.platforms.all;
    mainProgram = "reprepro";
  };
})
```

Don't use `with lib;`